SillyBusinessInc/SillyBusinessGame
# #112 Scene Available Doors manager

## Purpose of this PR:
The scenes have like 3 doors, and then on the scene load or something, then it should look at its own Room class and know how many doors should be vissible or not. And the door connection is relying on the table.

## How to test:
when the player enters the room the doors will appear in the scene depending on the connected room in the current scene(active room). And you can check the connected room in the door inspector. And you can see the connected room list in the console.

### links: 
https://github.com/SillyBusinessInc/SillyBusinessGame/issues/112
